RubberduckTests/Grammar/VBAParserTests.cs

```diff
@@ -2228,16 +2437,16 @@ End Type
             return Tuple.Create(parser, tree);
         }
 
-        private void AssertTree(VBAParser parser, ParserRuleContext root, string xpath)
+        private void AssertTree(VBAParser parser, ParserRuleContext root, string xpath, string message = "")
```

Comment:
Unused -> remove. At least that' d be my reasoning... then again we might as well make that parameter mandatory, because rn the message isn't in any way useful without the test. Make AV sing :)